test/plain-list/AutosuggestApp.test.js

```diff
@@ -140,13 +237,84 @@ describe('Plain list Autosuggest', () => {
       expectSuggestions([]);
     });
 
+    it('should not focus suggestions', () => {
+      expectFocusedSuggestion(null);
+    });
+
+    it('should clear the input when Escape is pressed', () => {
+      clickEscape();
+      expectInputValue('');
+    });
+  });
+
+  describe('when typing and matches do not exist (when selectFirstSuggestion is true)', () => {
```

Comment:
Again, there is a lot of repetition between this `describe` and the one above. Let's add only tests that are specific to the new feature (`selectFirstSuggestion = true`).